components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java

```diff
@@ -463,7 +463,7 @@ public void shutdown() {
         if (secMan != null) {
             secMan.checkPermission(new ManagementPermission("control"));
         }
-        log.info("Shutting down " + serverName + "...");
+        log.debug("Shutting down " + serverName + "...");
```

Comment:
We can check if debug is enabled

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/65/commits/51edbc2cd480c5bf084bbea1edb41444a02c0b18